docs/policy.md

```diff
@@ -13,6 +13,10 @@ A policy has following properties.
   you can specify target resource using "path" and "properties"
 - condition : additional condition (see below)
 - tenant_id : regexp matching the tenant, defaults to ``.*``
+- is_domain_owner: type of the token's scope.
```

Comment:
@morrisson these are the current ways to modify resource access:

1. schema/property/view
2. schema/property/permission
3. policy/effect
4. policy/properties vs. policy/blacklistProperties
5. policy/condition, where a user can specify additional SQL criteria based on custom schema properties
6. policy/condition/is_owner (besides, I initially thought that is_domain_owner is similar to this one)
7. operation/admin_role_as_super_user
8. is_domain_owner

To be honest, I think that this is already unmaintainable and impossible to reason about. For example, given a random property in a random schema, I could not image how a developer is supposed to tell if this property is visible/updateble to any given tenant without knowing the full system configuration.

In my opinion, we should break the backward compatibility (and break it hard, there's still 1.0.0 tag for any legacy users) and clean it once and for all. 

What do you think?

Reply:
Hmm, I do not agree on your points.

We should not include point 1 and point 7 in this discussion, point 1 is related to gohan_webui handling and point 7 is our internal.

Without point 2, it can be controlled in policies. That's enough simple and monolithic, IMO. In this sense, I can agree on that point 2 has a duplication of point 4 so removing point 2 might make it simpler.